specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/azureFirewall.json

```diff
@@ -383,6 +383,60 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/azureFirewalls/{azureFirewallName}/learnedIPPrefixes": {
+      "post": {
+        "operationId": "AzureFirewalls_ListLearnedPrefixes",
+        "description": "Retrieves a list of all IP prefixes that azure firewall has learned to not SNAT.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
```

Comment:
Consider moving towards definitions from v3 common-types: this is a perfect candidate. This wouldn't block ARM signoff, but would be a good change.

Reply:
We will address this in our next release

Reply:
OK, I won't block ARM signoff for this.